__tests__/api.test.js

```diff
@@ -136,4 +144,24 @@ describe("API tests", ()=>{
             })
         })
     })
+    describe("DELETE /api/comments/:comment_id", () => {
+        test("204: Deletes a comment by the comment ID", () => {
+            return request(app)
+            .delete("/api/comments/1")
+            .expect(204)
+        })
+        test("404: Check the comment was successfully deleted", () => {
+            return request(app)
+            .get(`/api/comments/1`)
+            .expect(404)
```

Comment:
You wouldn't want a 404 for a successful deletion. Just the status code 204 and no response body shows a successful deletion.